plutus-tx-plugin/test/Budget/9.2/constAccR.budget.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 1624333576
+| mem: 4909700})
```

Comment:
Compared to `constAccL`

```
({cpu: 1693471576
| mem: 5210300})
```

this is better.

I.e. right-folding wins over left-folding.